niworkflows/anat/mni.py

```diff
@@ -8,15 +8,20 @@
 import shutil
 import pkg_resources as pkgr
 
+from nilearn.plotting import plot_anat
 from nipype.interfaces.ants.registration import Registration, RegistrationOutputSpec
 from nipype.interfaces.base import (traits, isdefined, BaseInterface, BaseInterfaceInputSpec,
                                     File, InputMultiPath)
 
 from niworkflows.data import getters
+from niworkflows.common import report
 from niworkflows import __packagename__, NIWORKFLOWS_LOG
 
-MAX_RETRIES = 3
-
+DEFAULT_CUTS = {
```

Comment:
Ah, I see. And I presume we are confident that the reference image will almost always be MNI?